addOns/bugtracker/src/main/java/org/zaproxy/zap/extension/bugtracker/ExtensionBugTracker.java

```diff
@@ -32,7 +32,7 @@
 public class ExtensionBugTracker extends ExtensionAdaptor {
 
     public static final String NAME = "ExtensionBugTracker";
-    public Set<Alert> alerts = null;
+    Set<Alert> alerts = null;
```

Comment:
Can be removed the instance variable.

Reply:
It is actually used elsewhere.

Reply:
Right, but as mentioned previously https://github.com/zaproxy/zap-extensions/pull/2930#discussion_r632928289 it has no effect (the alerts are set again after).

Reply:
I'll have another try at it. I started getting exceptions when using the add-on last time I removed it.

Reply:
Remove the `initialize` call from the constructor (that's already called when the alerts are set).